src/styles/index.scss

```diff
@@ -1,3 +1,4 @@
 body {
   margin: 0;
+  box-sizing: border-box;
```

Comment:
it will not apply to all elements inside the body as box sizing is not inherited by default